Maquetado/scss/styles.scss

```diff
@@ -147,3 +149,21 @@ body {
 .footer {
   background-color: $blue;
 }
+
+@media (max-width: 1024px) {
+  .tools-container {
+    display: inline;
```

Comment:
What you want is the grid to have just 1 column, not change the whole display :smile: